### PR TITLE
Wonders 3

### PR DESCRIPTION
**Pull request by MemeWizard**
## Changelog:
- Changed Dazar'alor great work type from Aztec Pyramid to Royal Palace, since it's actually a Royal Palace that looks like Pyramid.
- Added Utgarde (Grand Fortress)
- Added Gundrak (Aztec Pyramid)
- Added Atal'Hakkar (Aztec Pyramid)

## Developer changelog:
- Added `religion_have_ritual_sacrifice_trigger` for changing `upgrade_sacrificial_altar` trigger and future usage.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2874,6 +2874,12 @@ nick_the_decrepit;the Decrepit;;;;;;;;;;;;;x
...
+wonder_gundrak_name_string;Gundrak;;;;;;;;;;;;;x
+wonder_gundrak_desc_string;Gundrak is a massive place, with thick walls, tall guard towers and heavy gates. Inside, the buildings are tiered and covered in steps, and wind about one another like a nest of small animals, each fitting into the next.;;;;;;;;;;;;;x
+wonder_atalhakkar_name_string;Atal'Hakkar;;;;;;;;;;;;;x
+wonder_atalhakkar_desc_string;Over a thousand years ago, the powerful Gurubashi Empire was torn apart by a massive civil war. An influential group of troll priests, known as the Atal'ai, attempted to bring back an ancient blood god named Hakkar the Soulflayer. Though the priests were defeated and ultimately exiled, the great troll empire buckled in upon itself. The exiled priests fled far to the north, into the Swamp of Sorrows. There they erected a great temple to Hakkar - where they could prepare for his arrival into the physical world. The great dragon Aspect, Ysera, learned of the Atal'ai's plans and smashed the temple beneath the marshes. However, it is believed that some of the fanatical Atal'ai may have survived Ysera's wrath - and recommitted themselves to the dark service of Hakkar.;;;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_zuwc_united_localisation.csv]: I wouldn't mention the pyramid destruction.

**Comment by zumbak04** [comment]
Giant KEK. I think we shouldn't add big wonder descriptions. Opened Atal'hakar, scrolled down, opened another wonder.
<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by MemeWizard** [comment]
@zumbak04, done!

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2874,6 +2874,12 @@ nick_the_decrepit;the Decrepit;;;;;;;;;;;;;x
...
 EVTDESC_WCDAC_8001;[Root.GetDWDevil]'s favours, [Root.GetSocietyRank] [Root.GetFirstName]\nThe Fel has given us so much, and now it is time for us to offer it a gift in return. Fel magic feeds on the vital energy of mortal creatures, and the council wants you to captivate and drain the soul of a mortal, [Root.GetSocietyRank]!\n[Root.GetDWExpression]\nSigned, [From.GetSocietyRank] [From.GetFirstName];;[Root.GetDWDevil]s Gunst, [Root.GetSocietyRank][Root.GetFirstName]\nDas Fel hat uns so viel gegeben, und jetzt ist es Zeit f�r uns, im Gegenzug ein Geschenk anzubieten. Die Fel-Magie n�hrt sich von der Lebensenergie sterblicher Kreaturen und der Rat m�chte, dass Ihr einen Sterblichen gefangen nehmt und ihm die Seele raubt,[Root.GetSocietyRank]!\n[Root.GetDWExpression]\nGezeichnet, [From.GetSocietyRank][From.GetFirstName];;;;;;;;;;;x
+wonder_utgarde_name_string;Utgarde Keep;;;;;;;;;;;;;x
+wonder_utgarde_desc_string;Utgarde Keep, a relic of a lost civilization among the central cliffs of the Howling Fjord.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
wonder_utgarde_desc_string;Utgarde Keep stands as a relic of a lost civilization along the central cliffs of the Howling Fjord.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 3]
```diff
@@ -2874,6 +2874,12 @@ nick_the_decrepit;the Decrepit;;;;;;;;;;;;;x
...
+wonder_gundrak_name_string;Gundrak;;;;;;;;;;;;;x
+wonder_gundrak_desc_string;Gundrak is a massive place, with thick walls, tall guard towers and heavy gates. Inside, the buildings are tiered and covered in steps, and wind about one another like a nest of small animals, each fitting into the next.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
wonder_gundrak_desc_string;Gundrak is a massive complex with thick walls, tall guard towers, and stalwart gates. Inside, its structures are tiered and laden in steps winding about one another like a nest of small animals, each fitting into the next.;;;;;;;;;;;;;x
```

**Pull request merged**